addons/crm_iap_lead_enrich/models/res_config_settings.py

```diff
@@ -1,16 +1,24 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import models, fields
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = "res.config.settings"
 
-    @api.onchange('lead_enrich_auto')
-    def _onchange_cron_lead_enrich(self):
-        """ change the active status of the cron according to the settings"""
-        if self.module_crm_iap_lead_enrich == True:
-            cron = self.sudo().with_context(active_test=False).env.ref('crm_iap_lead_enrich.ir_cron_lead_enrichment')
-            if cron:
-                cron.active = self.lead_enrich_auto != 'manual'
+    def _get_lead_enrich_auto(self):
```

Comment:
I would say: use a compute (not required, no default) and change inverse to set_values override :D .

Reply:
I first try with a compute, but it was not called when we open the settings page :/ (it depends on nothing and I think in this case we need to save the record to compute the value)

Reply:
You can use a false trigger. Indeed when being in "new" (create) mode you need a trigger or the field is not computed.

Reply:
And what about `get_values` so ? so we are consistent with `set_values` ?

Reply:
Why not. We could give it a try :)